pkg/features/features.go

```diff
@@ -61,6 +61,7 @@ const (
 )
 
 // FeatureGate is map of key=value pairs that enables/disables various features.
+// swagger:model FeatureGate
 type FeatureGate map[string]bool
```

Comment:
I wonder if we should expose that structure. A map is okay internally, but in the API it might be better to expose a new structure. With structure it is clear what feature flags are available and exposed, besides that we avoid exposing everything automatically.

Reply:
@zreigz WDYT about it?

Reply:
@kubermatic/sig-api WDYT about it?

Reply:
I agree with @maciaszczykm. You should create some API model for this